docs/layers/identity/aws-sso.mdx

```diff
@@ -134,63 +144,53 @@ It's important to note that the specifics of the integration process may be subj
     </Steps>
 
   </TabItem>
-</Tabs>
-
-
-## Supported External Identity Providers
-
-[Follow the AWS documentation for setting up an IdP integration with AWS](https://docs.aws.amazon.com/singlesignon/latest/userguide/supported-idps.html). This list includes Azure AD, CyberArk, JumpCloud, Okta, OneLogin, and Ping Identity. For other providers, see the next
-section.
 
-:::caution Integrating JumpCloud with AWS IAM Identity Center
+  <TabItem value="gsuite" label="GSuite and Other External IdPs">
+    <Intro>
+      For non-explicitly supported Identity Providers, such as GSuite, set up the app integration with a custom external
+      identity provider. The steps may be different for each IdP, but the goal is ultimately the same.
+    </Intro>
 
-The official AWS documentation for setting up JumpCloud with AWS IAM Identity Center is not accurate. Instead, please
-refer to the [JumpCloud official documentation](https://jumpcloud.com/support/integrate-with-aws-iam-identity-center)
+    :::tip aws-ssosync
```

Comment:
The `:::` Admonitions typically do not work inside of MDX tags. 

Instead, use:

```
import Admonition from '@theme/Admonition';

...

<Admonition type="tip" title="Use aws-ssosync">
...
</Admonition>
```